packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2528,7 +2532,7 @@ export interface SendBoxStylesProps extends BaseCustomStyles {
 
 // @public
 export interface StatefulCallClient extends CallClient {
-    createCallAgent(...args: Parameters<CallClient['createCallAgent']>): Promise<DeclarativeCallAgent>;
+    createCallAgent(tokenCredential: CommunicationTokenCredential, options?: CallAgentOptions, alternativeCallerId?: string): Promise<DeclarativeCallAgent>;
```

Comment:
Wondering if we can add `alternativeCallerId` to `options`?

Reply:
So adding an optional prop shouldn't be a breaking change since it doesn't break an end-user's implementation.
Also, we are not proposing that this function matches the headless function.

Reply:
okay sounds good, after writing the test I do like it better in the options than its own field. I will update this and the test then reach out to the ARB about this change. 

it will be needed if we are to store this prop in the callAgentState. otherwise it will have to go back to the original PR where it was in the callClientState